### ISSUE TITLE
Use hashids for event identifier

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-server#6668]
https://github.com/davidaurelio/hashids-python

This enables us to decode event ID without a database hit in future

**Comment by kushthedude** [comment]
@iamareebjamal Do we need to implement hashid in here https://github.com/fossasia/open-event-server/blob/496e2e2d69e30f23ebb62ec3caeedb2f7dd41ae4/app/models/event.py#L29 ?

**Comment by iamareebjamal** [comment]
Yes

**Comment by codedsun** [comment]
Taking this up!

**Comment by codedsun** [comment]
@iamareebjamal Do we need to remove the previous implementation to get the previous id or do we encode this id with hashid and where do we have to save this?

**Comment by iamareebjamal** [comment]
For now, this will just change the get_new_identifier function which will take id as input

**Comment by codedsun** [comment]
@iamareebjamal But how can we pass `id` here?

https://github.com/fossasia/open-event-server/blob/496e2e2d69e30f23ebb62ec3caeedb2f7dd41ae4/app/api/data_layers/EventCopyLayer.py#L47

and 

https://github.com/fossasia/open-event-server/blob/496e2e2d69e30f23ebb62ec3caeedb2f7dd41ae4/app/api/event_copy.py#L62

**Comment by iamareebjamal** [comment]
Yes, it needs to be thought about how identifier can be generated

**Comment by Anmollenka** [comment]
@codedsun is this issue still relevant?
